models/ConfigureForm.php

```diff
@@ -52,6 +53,15 @@ public function saveSettings()
         $settings->set('enabled', (boolean)$this->enabled);
         $settings->set('clientId', $this->clientId);
         $settings->set('clientSecret', $this->clientSecret);
+        
+        /* On saving, if no value is entered for orcidAttribute, then assign "orcid" to it.

```

Comment:
Use `//` comments please as using `/* */` is normally seen as disabling "dead code".